modules/ROOT/pages/relnotes-java-sdk.adoc

```diff
@@ -31,6 +31,61 @@ cluster management. The other dependency is RxJava which is both used
 internally and exposed as a first-class API to the user in order to
 perform scalable and performant asynchronous, reactive operations.
 
+== Version 2.7.0 (2 October 2018)
+
+Version 2.7.0 is the first stable release of the 2.7 series, bringing new
+features, enhancements and bugfixes over the last stable release.
+
+http://packages.couchbase.com/clients/java/2.7.0/Couchbase-Java-Client-2.7.0.zip[Download] | http://docs.couchbase.com/sdk-api/couchbase-java-client-2.7.0/[API
+Reference] | http://docs.couchbase.com/sdk-api/couchbase-core-io-1.7.0/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:2.7.0**
+* com.couchbase.client:**core-io:1.7.0**
+* io.reactivex:**rxjava:1.3.8**
+* io.opentracing:**opentracing-api:0.31.0**
+
+=== Enhancements & New Features
+
```

Comment:
Seems like we should have some prose here somewhere describing the new features rather than just have a laundry list.

Reply:
Done